source/reference/mongosqld.txt

```diff
@@ -1023,25 +1031,27 @@ with each.
      - ``mongosqld`` Options
      - Behavior
 
-   * - Standalone Reader
-     - - :option:`--sampleMode <mongosqld --sampleMode>` ``read``
-       - :option:`--sampleSource <mongosqld --sampleSource>` *not used*
+   * - Standalone Schema
+     - - :option:`--schemaMode <mongosqld --schemaMode>` ``custom``
+       - :option:`--schemaSource <mongosqld --schemaSource>` *not used*
      - :manual:`Standalone </reference/glossary/#term-standalone>`
        :program:`mongod` instance. :binary:`~bin.mongosqld`
```

Comment:
Not something you changed this time around, but this is actually unrelated to the MongoDB topology (perhaps "standalone" was a poor choice of terminology)